sdks/python/apache_beam/examples/fastavro_it_test.py

```diff
@@ -84,13 +84,11 @@ def record(i):
   }
 
 
-@unittest.skipIf(sys.version_info[0] == 3 and
-                 os.environ.get('RUN_SKIPPED_PY3_TESTS') != '1',
-                 'This test still needs to be fixed on Python 3'
-                 'TODO: BEAM-6522')
+@unittest.skip('Due to a known issue in avro-python3 package, this'
```

Comment:
Can we keep running this tests for Python 2 ? Otherwise this will become a regression in test coverage.

Reply:
Thanks for catching this, @chamikaramj. It's good to have a fresh pair of eyes on the PR. Note that in an earlier revision this test was skipped only on Python-3  https://github.com/apache/beam/commit/553cc73252c4ecf81f9a759e63d6d794753c57b3.
This is another argument in favor of NOT force-pushing squashed commits after a review iteration, but doing a squash before a final merge after a PR is approved.